src/components/AvatarGroup/AvatarGroupCounter.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import Tooltip from "../Tooltip/Tooltip";
+import Avatar from "../Avatar/Avatar";
+import PropTypes from "prop-types";
+import AvatarGroupCounterTooltipContainer from "./AvatarGroupCounterTooltipContainer";
+import cx from "classnames";
+import Counter from "../Counter/Counter";
+import styles from "./AvatarGroupCounter.module.scss";
+
+const AvatarGroupCounter = ({
+  counterTooltipAvatars,
+  counterProps,
+  counterTooltipCustomProps,
+  avatarSize,
+  avatarType
+}) => {
+  const {
+    color: counterColor = Counter.colors.LIGHT,
+    count: counterValue = counterTooltipAvatars.length,
+    prefix: counterPrefix = "+",
+    maxDigits: counterMaxDigits = 3
+  } = counterProps || {};
+
+  const counterSizeStyle = styles[avatarSize?.toString()];
+  const counterColorStyle = styles[counterColor];
+
+  if (!counterTooltipAvatars.length && !counterValue) {
+    return null;
+  }
+
+  return (
+    <AvatarGroupCounterTooltipContainer
+      avatars={counterTooltipAvatars}
+      counterTooltipCustomProps={counterTooltipCustomProps}
+      type={avatarType}
+    >
+      {/* eslint-disable jsx-a11y/no-noninteractive-tabindex */}
+      <div tabIndex={0} className={cx(styles.counterContainer, counterSizeStyle, counterColorStyle)}>
+        {/* eslint-enable jsx-a11y/no-noninteractive-tabindex */}
```

Comment:
I think we can delete this comment no?

Reply:
Need the comment to prevent eslint to complain about tabIndex on noninteractive element